specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json

```diff
@@ -10386,10 +10383,10 @@
         "type": "string"
       }
     },
-    "BlobObjectReplicationRules": {
+	"OrMetadata": {
```

Comment:
The indentation is little bit odd.
I'd rather call it ObjectReplicationMetadata.

Reply:
Can you take another look? Just updated